app/config/AppConfig.scala

```diff
@@ -93,11 +94,11 @@ class AppConfigImpl @Inject() (config: ServicesConfig, configuration: Configurat
 
   val apiGatewayContext: String = config.getString("api.gateway.context")
```

Comment:
Could group all the API config stuff together like it's done [here](https://github.com/hmrc/self-employment-business-api/blob/6b7e38c0da38e7b45a07d9a31612a1ada3fd5c2e/app/config/AppConfig.scala)